README.md

```diff
@@ -26,11 +26,20 @@ Virtual Environment setup:
 Running møte:
  - Start `memcached` by typing `memcached` in the terminal and pressing Return (Enter) key.
  - Edit [config.py](./files/config.py) with necessary information and place the `config.py` in [`mote/`](./mote)
- - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE`
+ - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE` (Optional)
    - You can do it by running the following command in command line too: `sed -i '/use_memcached = / s/False/True/' mote/config.py`
  - Enter the Pipenv's newly created virtualenv's shell with `pipenv shell`
  - Run `python runmote.py`
 
+OR
+
+Using `docker-compose` to run møte:
+ - Start `memcached` by typing `memcached` in the terminal and pressing Return (Enter) key.
+ - Edit [config.py](./files/config.py) with necessary information and place the `config.py` in [`mote/`](./mote)
```

Comment:
How about copying it within the container:
```
RUN cp ./files/config.py config.py
```
or adding a If statement, so, if the file exists we wouldn't execute it.
and then the RUN statements for sed.

Reply:
> How about copying it within the container:

You mean making the copy happen with the `Dockerfile`, right?

Reply:
I think we would want to keep only the
```
RUN cp ./files/config.py config.py
```
in the `Dockerfile` and leave other configurations to the user, for anyway once they are made - they should be copied to be actually some use.

Also, I believe that the destination for this copy should be `./mote/config.py` instead of just `config.py`.